packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -255,10 +251,6 @@ export class RichText extends Component {
 		} ).map( ( name ) => gutenbergFormatNamesToAztec[ name ] ).filter( Boolean );
 	}
 
-	onFormatChangeForceChild( record ) {
-		this.onFormatChange( record, true );
-	}
-
 	onFormatChange( record ) {
```

Comment:
@tug should we rename this to `onChange` to match the web side?

Reply:
We might try now! But I think it's outside the scope of this PR...

On the web `onChange` accepts a `record` and is going to apply it to the DOM tree to apply only the required modifications.
Since we don't use a DOM tree on native, we created our own version of `onChange`, one for value change (we kept the name `onChange` but we accept a native event argument instead) and one for format change (we called `onFormatChange`) in which we're calling `valueToFormat` to convert the record back into html to update the value instead of the dom.